otelcollector/test/test-cluster-yamls/configmaps/ama-metrics-settings-configmap.yaml

```diff
@@ -17,7 +17,7 @@ data:
     networkobservabilityRetina = true
     networkobservabilityHubble = true
     networkobservabilityCilium = true
-    prometheuscollectorhealth = true
```

Comment:
i think this was to test out the scenario when health is set to true, because there was an issue when fbit was crashing when health was set to true. Should we keep this as true?

Reply:
ok. didn't notice it. But if its test cluster's config map, may be we can keep it as true , to help debug issues /volumes ?

Reply:
sounds good, thanks